README.md

```diff
@@ -0,0 +1,4 @@
+# README
+
```

Comment:
The license and a brief description of the project would usually go here.

It doesn't need to be that long, three to four hundred words is probably enough.